CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change log
 
+## 0.11.0
+
+- **New**: Dynamic configuration for class-based clients! Class-based clients now accept a `Config` object in their constructor, allowing you to create multiple clients with different configurations on the fly.
```

Comment:
I'd like this to be part of version 0.10.0 so make sure both changelogs are updated to reflect that.